src/appkit.cs

```diff
@@ -24354,6 +24704,16 @@ interface NSAccessibility {
 #endif
 		[NullAllowed, Export ("accessibilityCustomActions", ArgumentSemantic.Copy)]
 		NSAccessibilityCustomAction [] AccessibilityCustomActions { get; set; }
+
+		[Mac (14, 0)]
+		[Abstract]
+		[Export ("accessibilityUserInputLabels", ArgumentSemantic.Copy)]
+		string [] AccessibilityUserInputLabels { get; set; }
+
+		[Mac (14, 0)]
+		[Abstract]
+		[Export ("accessibilityAttributedUserInputLabels", ArgumentSemantic.Copy)]
+		NSAttributedString [] AccessibilityAttributedUserInputLabels { get; set; }
```

Comment:
Isn't it a breaking change to add abstract members to an existing protocol?

Reply:
I missed that, you are correct.